frontend/app/_components/form.tsx

```diff
@@ -119,10 +119,14 @@ export default function Form(props: FormProps) {
       ))}
       {/* Global errors */}
       {globalFormErrors &&
-        globalFormErrors.map((error, index) => (
-          <span key={index} style={{ color: 'red' }}>
-            {error}
-          </span>
+        (typeof globalFormErrors === 'string' ? (
+          <span style={{ color: 'red' }}>{globalFormErrors}</span>
+        ) : (
```

Comment:
Oof, is a conditional the only way to do this? Not graceful but it works, nice fix

Reply:
Yeah I could/should normalize the data by making it an array of strings regardless, but before I get into it i'd like to have a clear map of how the API works (in terms of all the response possibilities), so I know how to build handlers based on that.